app/src/lib/stores/app-store.ts

```diff
@@ -575,13 +577,41 @@ export class AppStore extends TypedBaseStore<IAppState> {
   }
 
   private initializeZoomFactor = async () => {
-    const zoomFactor = await getCurrentWindowZoomFactor()
+    let zoomFactor = await getCurrentWindowZoomFactor()
     if (zoomFactor === undefined) {
```

Comment:
Shouldn't we check the zoom factor stored locally even if Chromium gives us no zoom factor? 🤔

Reply:
Technically it "shouldn't" be undefined. The undefined comes in to play because our browner window in the main process starts out as null. But, by the time this is called, it "should" always be available and therefore this should always return a value. Thus, the check is really just for typing sake. 

However.. after you said that, I realized this bit of logic could be slightly more succinct if I move that electron zoom factor fetching call into my checking method, and we get the double benefit of returning locally store if in some odd ball case in the future it did return undefined.